cmd/add_resource_test.go

```diff
@@ -75,12 +75,13 @@ func TestAddResourceExecution(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	client := mock.NewMockAddResourceClient(ctrl)
 
+	var emptyCreds meroxa.Credentials
+
 	r := meroxa.CreateResourceInput{
 		Type:        "postgres",
 		Name:        "",
 		URL:         "https://foo.url",
-		Credentials: nil,
-		Metadata:    nil,
+		Credentials: &emptyCreds,
```

Comment:
This will be a pointer to nil, is this what you want?

Reply:
@mwoods79 yeah... https://github.com/meroxa/meroxa-go/blob/master/resource.go#L34